### ISSUE TITLE
Typo in graph-types/isUnboundRelationship

### ISSUE THREAD

**Issue by zirkelc** [neo4j/neo4j-javascript-driver#679]
Hi guys,

I stumbled upon a typo in [graph-types.js](https://github.com/neo4j/neo4j-javascript-driver/blob/4.3/src/graph-types.js) for the method [isUnboundRelationship](https://github.com/neo4j/neo4j-javascript-driver/blob/4.3/src/graph-types.js#L256-L258):

```javascript
function isUnboundRelationship (obj) {
  return hasIdentifierProperty(obj, UNBOUNT_RELATIONSHIP_IDENTIFIER_PROPERTY)
}
```

The constant is misspelled as UNBOUN**T** instead of the actual definition of the [constant](https://github.com/neo4j/neo4j-javascript-driver/blob/4.3/src/graph-types.js#L29) as UNBOUN**D**:
```javascript
const UNBOUND_RELATIONSHIP_IDENTIFIER_PROPERTY = '__isUnboundRelationship__'
```

Best regards
Chris

**Comment by fbiville** [comment]
Thanks for the report! Would you be interested in submitting a patch?

**Comment by bigmontz** [comment]
Thanks for the reports, this patch for fix this issue is already done. 

This issue will be fixed in the version 4.3.
